static/js/server_events_dispatch.js

```diff
@@ -321,6 +319,17 @@ export function dispatch_normal_event(event) {
                     break;
             }
             break;
+
+        case "realm_default": {
+            let property_name = event.property;
+            if (event.property === "twenty_four_hour_time") {
+                property_name = "default_twenty_four_hour_time";
+            }
+            page_params["realm_" + property_name] = event.value;
```

Comment:
Hmm, what do you think about trying to put these in a `realm_default` object in the top-level of `page_params`, so that this would be just `page.params.realm_default[property_name] = event.value`?

Reply:
Yes, I though of doing a separate commit for this.